apps/style/common.scss

```diff
@@ -825,6 +825,49 @@ input[type="radio"] {
   font-size: large;
 }
 
+#puzzleRatingButtons {
+  hr {
+    border-top: 5px dotted #ccc;
+  }
+
+  p {
+    float: left;
+    font-size: 18px;
+    color: #9E9D9D;
+    margin-right: 10px;
+  }
+
+  svg {
+    pointer-events: all;
```

Comment:
according to http://caniuse.com/#feat=pointer-events pointer-events is not supported in IE < 11